src/python/pants/option/global_options.py

```diff
@@ -289,6 +289,12 @@ def register_bootstrap_options(cls, register):
                   'start up all concurrent invocations (e.g. in other terminals) without pantsd. '
                   'Enabling this option requires parallel pants invocations to block on the first')
 
+    # Calling pants command (inner run) from other pants command is not the intended behaviour.
```

Comment:
```suggestion
    # Calling pants command (inner run) from other pants command is unusual behaviour, and most users should never set this flag. It is automatically set by pants when an inner run is detected.
```

Saying "not intended behaviour" suggests to me that it shouldn't work, or that it's not allowed. Rather than that, let's explain why, and what the flag _is_ for.